gpu-operator/install-gpu-operator-proxy.rst

```diff
@@ -57,26 +57,18 @@ HTTP Proxy Configuration for Openshift
 ======================================
 
 For Openshift, it is recommended to use the cluster-wide Proxy object to provide proxy information for the cluster.
-Please follow the procedure described in `Configuring the cluster-wide proxy <https://docs.openshift.com/container-platform/4.8/networking/enable-cluster-wide-proxy.html>`_
+Follow the procedure described in `Configuring the cluster-wide proxy <https://docs.openshift.com/container-platform/4.8/networking/enable-cluster-wide-proxy.html>`_
```

Comment:
Let's use a non-versioned URL here. `latest` will automatically redirect us to the most recent stable release of OCP
```suggestion
Follow the procedure described in `Configuring the cluster-wide proxy <https://docs.openshift.com/container-platform/latest/networking/enable-cluster-wide-proxy.html>`_
```